aspnetcore/fundamentals/configuration/index.md

```diff
@@ -185,7 +185,7 @@ Configuration sources are read in the order that their configuration providers a
 
 A typical sequence of configuration providers is:
 
-1. Files (*appsettings.json*, *appsettings.&lt;Environment&gt;.json*, where `<Environment>` is the app's current hosting environment)
+1. Files (*appsettings.json*, *appsettings.{Environment}.json*, where `<Environment>` is the app's current hosting environment)
```

Comment:
- `<Environment>` --> `{Environment}`
- List the Azure Key Vault provider and link to our doc. That provider takes precedence over Secret Manager, since it's a more secure provider.

Reply:
> \<Environment> --> {Environment}

*DAH!* 🙈 Find and replace ... it's great ... NOT smart ... but great.